helm/hpcc/values.schema.json

```diff
@@ -763,6 +763,11 @@
           "description": "The default storage plane to write data files to",
           "type": "string"
         },
+        "storagePlanes": {
```

Comment:
@richardkchapman this isn't actually used - should I remove it, or leave it there for use by HPCC-25930?

Reply:
It's fine to leave.